docs/News.md

```diff
@@ -1,3 +1,49 @@
+    * [Endometrial Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ucec_tcga_gdc) *548 samples*
+    * [Esophageal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=esca_tcga_gdc) *186 samples*
+    * [Glioblastoma Multiforme (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=gbm_tcga_gdc) *610 samples*
+    * [Head and Neck Squamous Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hnsc_tcga_gdc) *530 samples*
+    * [Hepatocellular Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hcc_tcga_gdc) *379 samples*
+    * [High-Grade Serous Ovarian Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hgsoc_tcga_gdc) *602 samples*
+    * [Invasive Breast Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=brca_tcga_gdc) *1103 samples*
+    * [Lung Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=luad_tcga_gdc) *571 samples*
+    * [Lung Squamous Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=lusc_tcga_gdc) *503 samples*
+    * [Miscellaneous Neuroepithelial Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=mnet_tcga_gdc) *184 samples*
+    * [Non-Seminomatous Germ Cell Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=nsgct_tcga_gdc) *156 samples*
+    * [Pancreatic Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=paad_tcga_gdc) *186 samples*
+    * [Papillary Renal Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prcc_tcga_gdc) *292 samples*
+    * [Papillary Thyroid Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thpa_tcga_gdc) *515 samples*
+    * [Pleural Mesothelioma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=plmeso_tcga_gdc) *87 samples*
+    * [Prostate Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prad_tcga_gdc) *501 samples*
+    * [Rectal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=read_tcga_gdc) *171 samples*
+    * [Renal Clear Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ccrcc_tcga_gdc) *537 samples*
+    * [Soft Tissue (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=soft_tissue_tcga_gdc) *264 samples*
+    * [Stomach Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=stad_tcga_gdc) *443 samples*
+    * [Thymoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thym_tcga_gdc) *124 samples*
+    * [Uterine Carcinosarcoma/Uterine Malignant Mixed Mullerian Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ucs_tcga_gdc) *57 samples*
+    * [Uveal Melanoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=um_tcga_gdc) *80 samples*
+    * [Breast (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=breast_cptac_gdc) *154 samples*
```

Comment:
```suggestion
    * [Breast Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=breast_cptac_gdc) *154 samples*
```

Reply:
^ Niki had mentioned that he prefers "<body part> Cancer" instead of "<body part>", so I will be updating the names of the studies accordingly. See the discussion in \#data-feedback: https://cbioportal.slack.com/archives/C04PHKPKF2R/p1721693960892619?thread_ts=1721329662.398769&cid=C04PHKPKF2R